src/Views/AppInfoView.vala

```diff
@@ -76,8 +76,105 @@ namespace AppCenter.Views {
+                                // TRANSLATORS: Replace the link with the version for your language
+                                license_url = _("https://en.wikipedia.org/wiki/Public_domain");
+                            }
+                            break;
+                        case "free":
+                            // TRANSLATORS: Freedom, not price. See the GNU page.
+                            license_name = _("Free Software");
+                            if (license_url == null) {
+                                // TRANSLATORS: Replace the link with the version for your language
+                                license_url = _("https://www.gnu.org/philosophy/free-sw");
+                            }
+                            break;
+                        case "proprietary":
+                            license_name = _("Proprietary");
+                            break;
+                    }
+                } else {
+                    license_name = project_license;
+                    license_url = "https://choosealicense.com/licenses/";
 
+                    switch (project_license) {
+                        case "Apache-2.0":
+                            license_url = license_url + "apache-2.0";
+                            break;
+                        case "GPL-2":
+                        case "GPL-2.0":
+                        case "GPL-2.0+":
+                            license_url = license_url + "gpl-2.0";
+                            break;
+                        case "GPL-3":
+                        case "GPL-3.0":
+                        case "GPL-3.0+":
+                            license_url = license_url + "gpl-3.0";
+                            break;
+                        case "LGPL-2.1":
+                        case "LGPL-2.1+":
+                            license_url = license_url + "lgpl-2.1";
+                            break;
+                        case "MIT":
+                            license_url = license_url + "mit";
+                            break;
+                    }
+                }
+            }
+
+            var license_button = new Gtk.Button.with_label (license_name);
+            license_button.always_show_image = true;
+            license_button.image = new Gtk.Image.from_icon_name ("text-x-copying-symbolic", Gtk.IconSize.DND);
+            license_button.image_position = Gtk.PositionType.TOP;
+            license_button.tooltip_text = license_url;
+
+            unowned Gtk.StyleContext license_style_context = license_button.get_style_context ();
+            license_style_context.add_class (Gtk.STYLE_CLASS_DIM_LABEL);
+            license_style_context.add_class (Gtk.STYLE_CLASS_FLAT);
+
+            var noncurated_grid = new Gtk.Grid ();
+            noncurated_grid.margin_start = 24;
+            noncurated_grid.row_spacing = 3;
+            noncurated_grid.tooltip_markup = "<b>%s</b>\n%s\n%s".printf (
```

Comment:
If we're planning to keep the dialog (which I think we should), we should be able to vastly simplify this copy down. How about something like:

>Has not been reviewed by elementary for security, privacy, or system integration

Reply:
I agree sandboxing stuff can be split out, but I'd like to revisit that in a future branch and keep the messaging the same for now

Reply:
My issue is it feels like a *lot* to shove into a tooltip, especially when literally the same strings are gonna be shown again when they go to install. Is there a better way to simplify it here?

Reply:
Is there anything we can do to make it acceptable to merge into master making sure that this copy matches the dialog copy?

My general feeling is that we're probably going to iterate here a bit as we use it and I don't want to add a bunch of translation work unless we're really sure we want to diverge on these strings

Reply:
@danrabbit we could just use the same string as the dialog title for now, then iterate on the dialog (possibly removing it) and this copy later.